openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedTab/ActivityFeedTab.component.tsx

```diff
@@ -420,6 +495,29 @@ export const ActivityFeedTab = ({
             ),
             key: 'tasks',
           },
+          {
+            label: (
+              <div className="d-flex justify-between">
+                <Space align="center" size="small">
+                  <AnnouncementsIcon
+                    style={COMMON_ICON_STYLES}
+                    {...ICON_DIMENSION}
+                  />
+                  <span>{t('label.announcement-plural')}</span>
+                </Space>
+
+                <span>
+                  {!isUserEntity &&
+                    getCountBadge(
+                      countData.data.conversationCount,
+                      '',
+                      activeTab === ActivityFeedTabs.ANNOUNCEMENTS
+                    )}
+                </span>
+              </div>
+            ),
+            key: 'announcements',
+          },
         ]}
```

Comment:
we are using ActivityFeed in other places as well like Glossary and User page, can you check if we are reflecting the  announcement tab there as well ?

Reply:
Added check to prevent Announcements tab from being rendered in Glossary and Users Page.